Gemfile.lock

```diff
@@ -22,7 +22,7 @@ GIT
 
 GIT
   remote: https://github.com/openfoodfoundation/spree.git
-  revision: b7ad5b473f9e38c5a1882550b2b4e348f4824f1f
+  revision: f9264ebca0d8dd932a7dbee471514a3b85774c8f
```

Comment:
shouldn't this be https://github.com/openfoodfoundation/spree/pull/44/commits/73d0113dedbcb004ae457eb3f050a4855cc93931 instead?

Reply:
I used the merge commit: https://github.com/openfoodfoundation/spree/commits/2-1-0-stable/
I think it's the correct one right?

Reply:
yeah, right :+1: